tools/run_tests/build_artifact_python.sh

```diff
@@ -43,7 +43,11 @@ GRPC_PYTHON_USE_CUSTOM_BDIST=0  \
 GRPC_PYTHON_BUILD_WITH_CYTHON=1 \
 ${SETARCH_CMD} python setup.py  \
     bdist_wheel                 \
-    sdist                       \
+    sdist
```

Comment:
I question the utility of filing a bunch of bug-reports/feature-request issues for hacks that all fall under the umbrella of, "We're doing this to get a certain effect and don't care about how we get there." I am uncomfortable filing a bug report that implicitly says, "We're okay with this and we're going to support it."